simoc_server/util.py

```diff
@@ -134,4 +134,22 @@ def sum_attributes(objects, attribute_name):
     type of object.attribute
         The sum of all of the attributes
     """
-    return sum([x.__dict__[attribute_name] for x in objects])
\ No newline at end of file
+    return sum([x.__dict__[attribute_name] for x in objects])
+
+def to_volume(mass, density):
```

Comment:
I actually thought about what you asked the other day and realized that util wouldn't be a great place to put this, I actually made a agent_model_util for agent model specific helper functions.  I can move this over later